src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
@dabeng @dlabrecq I wrote a few more comments about the CSS below...let me know what you guys think?